Feature/better retail styling update

a-rousseau opened the pull request:
[AB#66620](https://dev.azure.com/orckestra001/da965986-c85b-43be-9915-219568b000e5/_workitems/edit/66620)
Fix alignment of icons in mobile
Adjust the padding of the navbar on mobile
Fix styling of the jumbotron heading font weight
Fix the header spacing by hiding the div with hidden content
Fix the row gap to take into account the product tile banners
Fix the layout of the product tiles to reflect best practices
Fix variants display on product tile
Removed the uppercase text styling classes from the buttons
Update font variable to be more generic and change the default font for the website to Raleway
Update the font files to use the 5.15.4 version of fontawesome
Remove References to v4 of fontawesome
Update the use of the shopping-cart to the shopping-bag for retail
change Header styling
change Menu Styling
Adjust icon sizes to be smaller
Fix search placeholder styling
Fix search box styling
Update the Home Page banners to use the new responsive advanced versions
Add classes .h1, .h2, .h3, .h4, .h5, .h6, .small to VisualEditor
Add block span to VisualEditor
Add missing headings blocks to VisualEditor

Commit:
Fix Header icon alignment in mobile

Fix alignment of icons in mobile
Adjust the padding of the navbar on mobile
Fix styling of the jumbotron heading font weight

Commit:
Merge branch 'dev' into feature/better-retail-styling-update

Commit:
Merge branch 'dev' into feature/better-retail-styling-update

Commit:
Fix grid and header spacing

Fix the header spacing by hiding the div with hidden content
Fix the row gap to take into account the product tile banners

Commit:
New Mobile product tile layout

Fix the layout of the product tiles to reflect best practices

Commit:
Fix Product Tile Variant display on mobile

Fix variants display on product tile

Commit:
Fix product tile style for mobile

Adjust variant alignment and positioning for product tiles on mobile

Commit:
Remove inline text styling classes on buttons

Removed the uppercase text styling classes from the buttons

Commit:
Fix product tile display

Fix styling on hover of the Product Tile in desktop mode
Hide the color variants when there is only one available color

Commit:
Remove console logs

Remove console logs from SearchResultsController.ts

Commit:
update font variable names

Update font variable to be more generic and change the default font for the website to Raleway

Commit:
update to fontawesome 5.15.4

Update fontawesome to version 5 to be able to use the shopping-bag icon

Commit:
Update implementation of fontawesome

Update the use of the shopping-cart to the shopping-bag for retail
Update the font files to use the 5.15.4 version of fontawesome
Remove References to v4 of fontawesome

Commit:
Update font styles and fix minicart button

Fix display of minicart button outside mobile
Fix font styling to use a different font for body and headings

Commit:
Fix add to cart button with badge styling

Fix position of the shopping bag icon
fix the position of the badge on the button

Commit:
Update the bootstrap card styling

Update the bootstrap card styling
Fix the card-header font size
Fix the inline card background color

Commit:
Update font styles

Update the font styles for the website

Commit:
Update Banners and Home page banner content

Update the banner options to include a full width version without any column classes
Update the styles for the banners

Commit:
Update HomePage Content Banners

Update the Home Page banners to use the new responsive advanced versions

Commit:
Make content larger on large desktops

Modify the container class on large desktops so it takes more width
Fix Product tile display to fit the new responsive width

Commit:
Add new format options to the Visual Editor tinymce

Add classes .h1, .h2, .h3, .h4, .h5, .h6, .small
Add block span
Add missing headings blocks

Commit:
Fix Header styling and Visual Editor format config file

Fixed malformed xml in common.xml for the Visual Editor
Changed the container width from lg to md to fit the styling defined for large desktops

Commit:
Update Mobile Menus Styling

Update mobile menu styling for the header
Update mobile menu styling for the footer
Add a new class to override the row-gap on mobile

Commit:
Update Mobile menu styling

add mobile menu row-gap fixes in main menu

Commit:
Update selected facet styling

Update the selected facets styling to represent more a pill list
Give a more disabled look to the pill facet

Commit:
Fix carousel mobile product tiles

Fix styling of carousel mobile product tiles
Fix add to cart button styling to reflect the rest of the website's buttons

Commit:
Fix Responsive Carousel Product Titles

Fix the styling of the product tiles in the Slick carousel for all device sizes

Commit:
Fix Header styling

change Header styling
change Menu Styling
Adjust icon sizes to be smaller
Fix search placeholder styling
Fix search box styling

Commit:
Update Header, titles, icons and pagination

Update logo in the header
Update the colors in the header
Update the dropdown to be completely full width
Fix footer styling on desktop
Change the arrow icons to chevrons
Adjust styling of mega menu
Adjust titles styling

Commit:
Update Product List / Search Page styling

Update button stylings to be coherent through the website
Reduce icon sizes
Adjust border colors

Commit:
Update Cart and Checkout page styles

Update styles for Order Summary
Update styles for promotion in order summary
Update styles for Discount display

Commit:
Fix My account stylings

Fix some stylings in the cards for Order Details
Fix store details page view

Commit:
Add new way to use Grid components

Add a Row component as a placeholder
Add a Column component independant from the Row and that can be made responsive by selecting different sizes for each breakpoint

Commit:
Merge branch 'dev' into feature/better-retail-styling-update

a-rousseau commented:
![Screen Shot 2022-10-24 at 2 05 46 PM](https://user-images.githubusercontent.com/44198946/197595640-6019e9d0-51f9-4254-a55a-c18747677a71.png)
![Screen Shot 2022-10-24 at 2 06 03 PM](https://user-images.githubusercontent.com/44198946/197595656-c8b2ecb2-a723-44a8-85e2-30f3e4744710.png)
![Screen Shot 2022-10-24 at 2 06 17 PM](https://user-images.githubusercontent.com/44198946/197595663-4403e305-c49a-43b4-aa62-637c978d2f0f.png)
![Screen Shot 2022-10-24 at 2 06 33 PM](https://user-images.githubusercontent.com/44198946/197595669-810bb6dd-8524-4d91-83fc-897c0bd0e882.png)
![Screen Shot 2022-10-24 at 2 06 50 PM](https://user-images.githubusercontent.com/44198946/197595673-24dc1105-3f76-4f3f-86f1-ca1540fe7232.png)
![Screen Shot 2022-10-24 at 2 07 03 PM](https://user-images.githubusercontent.com/44198946/197595676-c71b182d-42e2-4ed2-b6c3-3044f41964d8.png)
![Screen Shot 2022-10-24 at 2 08 31 PM](https://user-images.githubusercontent.com/44198946/197595679-95468fc4-e554-4d6b-98af-cee8085a181e.png)

InnaBoitsun commented on src/Orckestra.Composer.C1.Content.EN-CA/Package/Datas/Composite.Data.Types.IPagePlaceholderContents_public.xml:
```diff
@@ -43,8 +43,7 @@
   <Add PageId="f3dbd28d-365f-4d3e-91c3-7b730b39b294" PlaceHolderId="pageHeader" Content="&lt;html xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&#xA;&#x9;&lt;head&gt;&#xA;&#x9;&#x9;&lt;f:function xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot; name=&quot;Composer.Header.PageHeader&quot; /&gt;&#xA;&#x9;&lt;/head&gt;&#xA;&#x9;&lt;body&gt;&#xA;&#x9;&lt;/body&gt;&#xA;&lt;/html&gt;" C ...
-  <Add PageId="f3dbd28d-365f-4d3e-91c3-7b730b39b294" PlaceHolderId="body" Content="&lt;html xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&#xA;&#x9;&lt;head&gt;&#xA;&#x9;&lt;/head&gt;&#xA;&#x9;&lt;body&gt;&#xA;&#xA;&lt;f:function name=&quot;Composer.Banners.HeroBanner&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot;&gt;&lt;f:param name=&quot;BackgroundImage&quot; value=&quot ...
-
+<Add PageId="f3dbd28d-365f-4d3e-91c3-7b730b39b294" PlaceHolderId="body" Content="&lt;html xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&#xA;&#x9;&lt;head&gt;&#xA;&#x9;&lt;/head&gt;&#xA;&#x9;&lt;body&gt;&#xA;&#xA;&lt;f:function name=&quot;Composer.Banners.HeroBanner&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot;&gt;&lt;f:param name=&quot;BackgroundImage&quot; value=&quot;M ...
```
clean VersionId

InnaBoitsun commented on src/Orckestra.Composer.C1.Content.EN-CA/Package/Datas/Composite.Data.Types.IPagePlaceholderContents_administrated.xml:
```diff
@@ -43,8 +43,7 @@
   <Add PageId="f3dbd28d-365f-4d3e-91c3-7b730b39b294" PlaceHolderId="pageHeader" Content="&lt;html xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&#xA;&#x9;&lt;head&gt;&#xA;&#x9;&#x9;&lt;f:function xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot; name=&quot;Composer.Header.PageHeader&quot; /&gt;&#xA;&#x9;&lt;/head&gt;&#xA;&#x9;&lt;body&gt;&#xA;&#x9;&lt;/body&gt;&#xA;&lt;/html&gt;" C ...
-  <Add PageId="f3dbd28d-365f-4d3e-91c3-7b730b39b294" PlaceHolderId="body" Content="&lt;html xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&#xA;&#x9;&lt;head&gt;&#xA;&#x9;&lt;/head&gt;&#xA;&#x9;&lt;body&gt;&#xA;&#xA;&lt;f:function name=&quot;Composer.Banners.HeroBanner&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot;&gt;&lt;f:param name=&quot;BackgroundImage&quot; value=&quot ...
-  <Add PageId="0ac9d403-ac27-468e-94ac-9f70bfefd063" PlaceHolderId="pageHeader" Content="&lt;html xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&#xA;&#x9;&lt;head&gt;&#xA;&#x9;&#x9;&lt;f:function xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot; name=&quot;Composer.Store.PageHeader&quot; /&gt;&#xA;&#x9;&lt;/head&gt;&#xA;&#x9;&lt;body&gt;&#xA;&#x9;&lt;/body&gt;&#xA;&lt;/html&gt;" Ch ...
+<Add PageId="f3dbd28d-365f-4d3e-91c3-7b730b39b294" PlaceHolderId="body" Content="&lt;html xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&#xA;&#x9;&lt;head&gt;&#xA;&#x9;&lt;/head&gt;&#xA;&#x9;&lt;body&gt;&#xA;&#xA;&lt;f:function name=&quot;Composer.Banners.HeroBanner&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot;&gt;&lt;f:param name=&quot;BackgroundImage&quot; value=&quot;M ...
```
clean VersionId

InnaBoitsun commented on src/Orckestra.Composer.Website/App_Data/Razor/Composer/SingleCheckout/OrderSummary.cshtml:
```diff
@@ -17,14 +17,14 @@
                 </h3>
-                <div class="card-body  bg-light" v-cloak="true">
-                    @OrderSummaryHelpers.Rewards()
+                @OrderSummaryHelpers.Rewards()
```
you removed v-cloak="true" and now it looks like this:
![AZ6pUESZmT](https://user-images.githubusercontent.com/6649972/197793909-08828300-c307-480c-8a1f-4ff4711a354b.gif)
you can add it back inside helper  @OrderSummaryHelpers.Rewards()

a-rousseau replied:
Oops, will do

Commit:
Fix comments

Remove VersionId from content pages
Fix OrderSummary display on checkout page to be hidden when fetching data
